src/main/java/org/example/utility/UtilityMethods.java

```diff
@@ -37,4 +37,14 @@ public static int countEven(int[] array) {
         }
         return evenElements;
     }
+    public static int countPositive(int [] array){
+        checkNotNull(array,"Your array is empty.");
+        int countPositive = 0;
+        for (int i : array) {
+            if(i > 0){
```

Comment:
Please add a space after `if` and before `{`.  

FYI: See "[Google Java Style Guide: 4.6.2 Horizontal whitespace](https://google.github.io/styleguide/javaguide.html#s4.6.2-horizontal-whitespace)"